observer/observer.go

```diff
@@ -0,0 +1,96 @@
+package observer
+
+import (
+	"strconv"
+
+	"github.com/letsencrypt/boulder/cmd"
+	blog "github.com/letsencrypt/boulder/log"
+
+	_ "github.com/letsencrypt/boulder/observer/probers/dns"
+	_ "github.com/letsencrypt/boulder/observer/probers/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	statMonitors = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "obs_monitors",
+			Help: "details of each configured monitor",
+		},
+		[]string{"name", "kind", "valid"},
+	)
+	statObservations = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "obs_observations",
+			Help:    "details of each probe attempt",
+			Buckets: []float64{.001, .002, .005, .01, .05, .1, .5, 1, 2, 5, 10},
```

Comment:
feels odd that this list of buckets goes 1, 2, 5, 1, 5, 1, 5, 1, 2, 5 (why are the hundredths and tenths sections missing '2' buckets?).

Reply:
Jacob and I went over this together and I just went with his suggestion outright. I'll admit that's a bit cargo-cultish of me; I'll be sure to revisit this before marking the comment resolved :)

Reply:
Yeah, I just proposed some buckets while live-reviewing that seemed a little better. They are probably not optimal.